src/internal/connector/exchange/attachment.go

```diff
@@ -95,24 +94,26 @@ func uploadLargeAttachment(
 	uploader attachmentUploadable,
 	attachment models.Attachmentable,
 ) error {
-	ab := attachmentBytes(attachment)
-	size := int64(len(ab))
+	var (
+		bs   = attachmentBytes(attachment)
+		size = int64(len(bs))
+	)
 
-	session, err := uploader.uploadSession(ctx, *attachment.GetName(), size)
+	session, err := uploader.uploadSession(ctx, ptr.Val(attachment.GetName()), size)
```

Comment:
not this PR per se, but do we need to check for an empty attachment name?

Reply:
Good question: I assume not, based on the prior code?  But can't say for sure.  @dadams39 would know better.